wpilibc/src/main/native/include/frc/motorcontrol/Victor.h

```diff
@@ -1,45 +0,0 @@
-// Copyright (c) FIRST and other WPILib contributors.
-// Open Source Software; you can modify and/or share it under the terms of
-// the WPILib BSD license file in the root directory of this project.
-
-#pragma once
-
-#include "frc/motorcontrol/PWMMotorController.h"
-
-namespace frc {
-
-/**
- * Vex Robotics %Victor 888 Motor %Controller.
- *
- * The Vex Robotics %Victor 884 Motor %Controller can also be used with this
- * class but may need to be calibrated per the Victor 884 user manual.
- *
- * Note that the %Victor uses the following bounds for PWM values.  These
- * values were determined empirically and optimized for the %Victor 888. These
- * values should work reasonably well for %Victor 884 controllers as well but
- * if users experience issues such as asymmetric behavior around the deadband
- * or inability to saturate the controller in either direction, calibration is
- * recommended. The calibration procedure can be found in the %Victor 884 User
- * Manual available from Vex.
```

Comment:
Are we concerned about losing the details of this doc comment?

Reply:
I doubt any teams use Victor 884 motor controllers anymore.

Reply:
Victor 884s were discontinued before current FRC students started grade school

Reply:
> Victor 884s were discontinued before current FRC students started grade school
🤮 I'm old

Reply:
I suppose that brings up a secondary question (for a different PR), if this should still be included in the library

Reply:
Victor 884s and 888s are still legal according to R504 in the 2024 game manual, and the PWM motor controller classes cost almost nothing to maintain (even less now with this PR). I'm inclined to keep them until they're made illegal.